docs/Users_Guide/getting_started.rst

```diff
@@ -0,0 +1,356 @@
+***************
+Getting Started
+***************
+
+This chapter reviews some important things to consider before starting
+a METplus project.
+
+Questions to Consider
+=====================
+
+Questions to ask when applying METplus to a new testing and evaluation project
+------------------------------------------------------------------------------
+
+If a user is new to developing a verification or evaluation system or
```

Comment:
This reads a little bumpy -- maybe "If a user is new to developing a verification or evaluation system or **specifically** to using METplus", or "If a user is new to **the concept of developing a verification or evaluations system**, there are...". Bolded for changes, not to be bolded in the actual text!

Reply:
Thanks, John.  I went with "If a user is new to the concept of developing a verification or evaluations system, there are...".